### PR TITLE
Updating to bootstrap4

### PR DESCRIPTION
**Pull request by VickySteeves**
View live: https://vickysteeves.github.io/reprozip/index.html

I finished moving this site to bootstrap4 and making it consistent with reprozip-examples. I have a few more things to do before this is ready for review:

- [x] Add graphics to about page on reprozip showing how it works
- [x] menion reproserver somewhere?
- [x] Fix sizing on news table 
- [x] Update news to current stuff going on (it was last updated in like 2017)

### CODE REVIEW

**Comment by julianafreire** [comment]
Looks cool.

We should add an acknowledgment to the Moore and Sloan Foundations: The development of ReproZip has been supported by the Moore and Sloan Foundations (add link http://msdse.org), under the NYU Moore-Sloan Data Science Environment (add link https://cds.nyu.edu/mooresloan)

Few suggestions:
ReproZip allows you to pack your research along with all necessary data files --> Should we mention it does so automatically?  E.g., ReproZip allows you to pack your research along with all necessary data files, and it does so by automatically tracking all dependencies. Using ReproZip, with a single command, you can quickly create virtual machines and containers.


The development of ReproZip was always been --> has always been
We encourage a transparent process to eliminate the black-box of development --> remove this sentence
We encourage folks to submit pull requests... folks --> users and developers (?)

from desktop applications to scripts in various languages to dynamic websites to high performance computing experiments--> from desktop applications and scripts in various languages to dynamic websites to high performance computing experiments

 You can then reproduce your work across various OS's! --> remove the exclamation mark

**Comment by julianafreire** [comment]
I would split  https://vickysteeves.github.io/reprozip/news.html into two pages, one for news and
one for Publications and Presentations

**Comment by VickySteeves** [comment]
Thanks for the feedback, @julianafreire ! I made all the updates and I think this is in good shape to be merged in, barring changes from @remram44 or @fchirigati.

File: index.html [hunk 1]
```diff
@@ -38,244 +37,137 @@
...
-$ reprounzip vagrant <span class="s">run</span> mydirectory
-<span class="nt">experiment: 0%... 25%... 50%... 75%... 100%
-result: 42.137</span>
-$ reprounzip vagrant <span class="s">upload</span> /tmp/new_config:global-config
-$ reprounzip vagrant <span class="s">run</span> mydirectory --cmdline <span class="na">./myexperiment --other --options inputs/somefile.csv</span>
-<span class="nt">experiment: 0%... 25%... 50%... 75%... 100%
-result: -17.814</span></code></pre>
-              </div>
+            <div class="card-footer">
+              <a class="btn btn-outline-primary" href="#" role="button"> Try out ReproZip today &raquo;</a>
```
Comment (remram44) [inline index.html]: This link doesn't go anywhere

File: css/custom.css [hunk 2]
```diff
@@ -1,252 +1,98 @@
+@import url('../webfonts/Montserrat');
```
Comment (remram44) [inline css/custom.css]: I don't think this is right

Comment (VickySteeves) [reply css/custom.css]: Why not? The font is loading for you, isn't it?

File: index.html (hunk 1 above)
Comment (VickySteeves) [reply index.html]: Good catch! I will fix this to go to the right place.

File: css/custom.css (hunk 2 above)
Comment (remram44) [reply css/custom.css]: No I just see the default sans-serif.

The console shows:
> Blocked loading mixed active content “http://vickysteeves.github.io/reprozip/webfonts/Montserrat/”

And that URL is a 404 HTML page: https://vickysteeves.github.io/reprozip/webfonts/Montserrat/

**Comment by VickySteeves** [comment]
I fixed those two issues from @remram44 in this last commit. Thanks for the review Remi & Juliana!

File: css/custom.css [hunk 3]
```diff
@@ -1,252 +1,98 @@
...
+.nav-item.active {
+	background-color: blue;
 }
-.navbar-inverse .navbar-nav > li > a:hover,
-.navbar-inverse .navbar-nav > li > a:focus {
-  color: #ffffff;
-  background-color: transparent;
+
+.nav.navbar-nav.ml-auto > .nav-item > .nav-link {
+  font-size:2em;
```
Comment (remram44) [inline css/custom.css]: This seems to be the difference between the two. It causes the social icons to have a different size, and the overall navbar too.

**Pull request merged**